src/extra/flash/info

```diff
@@ -0,0 +1,2 @@
+author="Atirut Wattanamongkol"
+description"Flash your EEPROM with your own BIOS"
```

Comment:
syntax error here - should be `description="Flash your EEPROM with your own BIOS"`